spec/lib/msf/ui/console/command_dispatcher/core_spec.rb

```diff
@@ -397,4 +397,96 @@ def set_tabs_test(option)
       end
     end
   end
+
+  describe "#cmd_sessions" do
+    before(:each) do
+      allow(driver).to receive(:active_session=)
+      
+    end
+
+    context "with no sessions" do
+      it "should show an empty table" do
+        core.cmd_sessions
+        expect(@output).to eq([
```

Comment:
It'd be great to be consistent with the use of `match_table` for these newly added tests :+1:

Reply:
Looked into this a bit, this case doesn't return a table so using match table results in failure

Reply:
`match_table` is a misnomer, it's mostly just a `match_multiline_string` matcher

Reply:
Interesting. There does seem to be some sort of formatting mismatch with the expecation though.
```
  1) Msf::Ui::Console::CommandDispatcher::Core#cmd_sessions with no sessions should show an empty table
     Failure/Error:
               expect(@output).to match_table <<~TABLE
                 Active sessions
                 ===============

                 No active sessions.
                 TABLE

       Expected:
       'Active sessions'
       '==============='
       ''
       'No active sessions.'
       Received:
       '["Active sessions", "===============", "", "No active sessions."]'
       Raw Result:
       ["Active sessions", "===============", "", "No active sessions."]

       Diff:
       @@ -1,5 +1,2 @@
       -Active sessions
       -===============
       -
       -No active sessions.
       +["Active sessions", "===============", "", "No active sessions."]
```
I'll dive deeper and see what I can uncover

Reply:
It looks like you're expecting the string to equal an array:
```
       Expected:
       'Active sessions'
       '==============='
       ''
       'No active sessions.'
       Received:
       '["Active sessions", "===============", "", "No active sessions."]' <----
```

i.e. you're missing the `@output.join("\n")` that's being used in the other test examples

Reply:
Yep, thank ya!